en-US/snippets/2017/mar2017.lang

```diff
@@ -0,0 +1,87 @@
+## NOTE: reference bug 1336204; Deadline: 24 February, 2017
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Link for Google Play: https://play.google.com/store/apps/details?id=org.mozilla.firefox&amp
+## NOTE: Link for App Store: https://itunes.apple.com/app/apple-store/id989804926?mt=8
+## NOTE: Campaign: Mobile Promotions
+
+
+## MAX_LENGTH: 120 characters
```

Comment:
I confess I'm quite confused by these max-length. In the bug you wrote that `Under Mobile Promotions section, the headers have character limitation of 120.`. Can you confirm the limitation is only for the first line of the snippet? 120+180 seems like a huge size, especially when English is a mere 33 characters.

Even with that clarified, it's unclear why only this one has the 120 characters limit set.

One more thing: when you add a limit with `## MAX_LENGTH`, you also need to add a comment for localizers to tell them that. That's valid for the entire file.